readme.md

```diff
@@ -483,13 +484,16 @@
 - [Non-Financial Blockchain](https://github.com/machinomy/awesome-non-financial-blockchain) - Non-financial blockchain applications.
 - [Mastodon](https://github.com/tleb/awesome-mastodon) - Open source decentralized microblogging network.
 
+## Conferences, Meetups
+- [Conferences](https://github.com/RichardLitt/awesome-conferences)
```

Comment:
This entry is not part of the pull request, not mentioned in the commit. According to [guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md#adding-to-this-list) each PR should have a single entry.